diff -Naur edk2/MdeModulePkg/MdeModulePkg.dsc edk2-patch/MdeModulePkg/MdeModulePkg.dsc
--- edk2/MdeModulePkg/MdeModulePkg.dsc	2021-08-14 13:00:42.596456809 -0400
+++ edk2-patch/MdeModulePkg/MdeModulePkg.dsc	2021-08-14 13:43:34.681539758 -0400
@@ -495,6 +495,9 @@
   MdeModulePkg/Universal/RegularExpressionDxe/RegularExpressionDxe.inf
   MdeModulePkg/Universal/SmmCommunicationBufferDxe/SmmCommunicationBufferDxe.inf
   MdeModulePkg/Universal/Disk/RamDiskDxe/RamDiskDxe.inf
+  
+[Components]
+  ../../boot/mach/efi/nbbuild.inf
 
 [Components.X64]
   MdeModulePkg/Universal/CapsulePei/CapsuleX64.inf
